x-pack/platform/plugins/shared/inference/server/chat_complete/anonymization/regex_worker_wrapper.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// eslint-disable-next-line @kbn/imports/no_boundary_crossing
+require('../../../../../../../../src/setup_node_env');
```

Comment:
🙈

Reply:
I removed it in favor of `require('ts-node')`